R/hideToast.R

```diff
@@ -0,0 +1,28 @@
+
+
+#' Hide existing toast messages
+#'
+#' @param animate a logical indicating whether to remove the toast message(s) instantly or use its \code{hideMethod} with animations to remove (Default).
+#' @param session the Shiny session. Defaults to \code{shiny::getDefaultReactiveDomain()}.
+#'
+#' @export
+#'
+#' @importFrom shiny getDefaultReactiveDomain
+#'
+#' @return `invisible()`
+#'
+hideToast <- function(
+  animate = FALSE,
```

Comment:
Shouldn't this be TRUE by default?

Reply:
That's my fault on the docs. Had default as FALSE when it was `removeToast` because `toastr.remove()` doesn't use animation. But now that it's `hideToast`, I'm going to make it default to TRUE. I think that's better functionality anyway